lib/legacy/polymer.dom.html

```diff
@@ -282,6 +282,7 @@
 
     /**
      * Returns the `composedPath` for this event.
+     * @return {!Array<Node>}
```

Comment:
@aomarks do you know why these don't turn up in the types? the getters. also, this should be `EventTarget` in typescript but closure doesn't seem to know what `EventTarget` is, so i set it as `Node` for now...